internal/pkg/config/config.go

```diff
@@ -46,7 +50,7 @@ func GetDefaultConfig() *Config {
 		Path: "default",
 		Fields: []Field{{
 			Title:      "Time",
-			Kind:       FieldKindTime,
+			Kind:       FieldKindNumericTime,
```

Comment:
I added it as the default in the readme, and listed the different types for time. I feel like it would probably be helpful for the readme to have a subsection on the different time types. What are your thoughts on that?

Reply:
It is a good idea, also it would be good to extend `assets/example.log` with more time examples.

`make run` points to this file for a quick preview.